moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -43,6 +43,7 @@ namespace
 {
 // The threshold above which `override_velocity_scaling_factor` will be used instead of computing the scaling from joint bounds.
 const double SCALING_OVERRIDE_THRESHOLD = 0.01;
+const double MIN_ANGLE_THRESHOLD = 1E-16;
```

Comment:
Add a comment to this parameter.

Also, can these two things be converted to `constexpr` instead?